### ISSUE TITLE
Add timings to Food Center

### ISSUE THREAD

**Issue by kanaiah** [ZeroFood/ZeroFood-Backend#2]
Currently, we are not storing opening and closing times of food center. @anoopmunshi , how do you want to store the timings for each food center? Can a food center have multiple timings?

**Comment by vikoka** [comment]
Yes @kanaiah , It will have closing and opening times. I have updated the Front-end with changes for from and to timing. for lunch and dinner in the latest push. see the list of changes done with latest commit and also I don't think we will have breakfast in the scenario. Further,  we should allow "No Mentioned" if there is no input or user leaves the field blank. 

Here is the Image for reference of  HTML latest commit ![image](https://user-images.githubusercontent.com/14233161/78531576-51c01800-7803-11ea-8c01-5123b8e35a58.png)
